micromamba.el

```diff
@@ -89,6 +116,17 @@ Pass ARGS as arguments to the program."
     (goto-char (point-min))
     (json-read)))
 
+(defvar micromamba--config nil
+  "Cached copy of configuration that Micromamba sees (including `condarc', etc).
+Set for the lifetime of the process.")
+
+(defun micromamba--get-config()
+  "Return current Conda configuration.  Cached for the lifetime of the process."
+  (if (not (eq micromamba--config nil))
+      micromamba--config
+    (let ((cfg (micromamba--call-json "config" "list" "--json")))
+      (setq micromamba--config cfg))))
```

Comment:
I suggested removing that above, but this can be written more concisely as follows:
```lisp
(or micromamba--config
    (setq micromamba-config
          (micromamba--call-json "config" "list" "--json")))
```